src/otx/api/entities/image.py

```diff
@@ -91,7 +92,9 @@ def numpy(self) -> np.ndarray:
             np.ndarray: NumPy representation of the image.
         """
         if self.__data is None:
-            return cv2.cvtColor(cv2.imread(self.__file_path), cv2.COLOR_BGR2RGB)
+            img = PILImage.open(self.__file_path)
```

Comment:
From my experience, OpenCV can read even corrupted images, so this change reduces coverage. Could we consider a fallback to `imread` here?

Reply:
Thank you for the response. Let me add a fallback logic.

Reply:
Here is the fix commit 85de6ba8db68488e5b104de8179f4f8e07d86b53